### PR TITLE
[14.0][MIG] account_statement_import_online_qonto

### PR DESCRIPTION
**Pull request by alexis-via**
Ready for review/merge

### CODE REVIEW

**Comment by alexis-via** [comment]
Travis is green now

**Review by rafaelbn — APPROVED** [review]
Thank you! 👍 could you please review error message?

**Review by StefanRijnhart — APPROVED** [review]

**Comment by StefanRijnhart** [comment]
@alexis-via still the same error
```
docutils.utils.SystemMessage: /tmp/tmpqjvu0ehr/account_statement_import_online_qonto/README.rst:3: (WARNING/2) Duplicate explicit target name: "qonto".
```

File: account_statement_import_online_qonto/README.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Will be regenerated from the readme subdir
```
Comment (pedrobaeza) [inline account_statement_import_online_qonto/README.rst:1]: Don't erase the README. This makes double diff and increase uselessly the size of the repo: one for removing content, and the other for adding it again.

File: account_statement_import_online_qonto/readme/CONFIGURE.rst [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+#. Open the Qonto bank journal.
+#. Set *Bank Feeds* to *Online (OCA)*.
+#. Select *Qonto.com* as online bank statements provider in
+   *Online Bank Statements (OCA)* section.
+#. Save.
+#. Click on the provider and configure provider-specific settings.
+
+To obtain *Login* and *Key*:
+
+#. Connect to the `Qonto <https://app.qonto.com/>`_ website and authenticate.
```
Comment (pedrobaeza) [inline account_statement_import_online_qonto/readme/CONFIGURE.rst]:
```suggestion
#. Connect to the `Qonto <https://app.qonto.com/>`__ website and authenticate.
```

File: account_statement_import_online_qonto/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1 @@
+This module provides online bank statements from `Qonto <https://qonto.com/>`_ via their `API <https://api-doc.qonto.com/>`_.
```
Comment (pedrobaeza) [inline account_statement_import_online_qonto/readme/DESCRIPTION.rst]:
```suggestion
This module provides online bank statements from `Qonto <https://qonto.com/>`__ via their `API <https://api-doc.qonto.com/>`__.
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
And please use the icon I have put in 13.0 version: https://github.com/OCA/bank-statement-import/blob/13.0/account_bank_statement_import_online_qonto/static/description/icon.png

**Comment by pedrobaeza** [comment]
@alexis-via are you going to fix this?

**Comment by victoralmau** [comment]
Please, cherry-pick https://github.com/OCA/bank-statement-import/pull/530

**Comment by alexis-via** [comment]
As you know, I plan to re-develop account_statement_import_online, so I won't invest time in this module any more.

**Pull request closed without merging**